core/src/main/java/jenkins/install/SetupWizard.java

```diff
@@ -161,6 +163,11 @@ public HttpResponse doCompleteInstall() throws IOException, ServletException {
      * This filter will validate that the security token is provided
      */
     private final Filter FORCE_SETUP_WIZARD_FILTER = new Filter() {
+        private Pattern RESOURCE_PATTERN = Pattern.compile(".*[.](css|ttf|gif|woff|eot|png|js)");
```

Comment:
But non-static may cause some performance degradation due to the frequent compilation in tests [and real life?]